eagle-jpm/eagle-jpm-util/src/main/java/org/apache/eagle/jpm/util/jobrecover/RunningJobManager.java

```diff
@@ -49,12 +53,21 @@ private CuratorFramework newCurator(String zkQuorum, int zkSessionTimeoutMs, int
         );
     }
 
-    public RunningJobManager(String zkQuorum, int zkSessionTimeoutMs, int zkRetryTimes, int zkRetryInterval, String zkRoot) {
+    public RunningJobManager(String zkQuorum, int zkSessionTimeoutMs, int zkRetryTimes, int zkRetryInterval, String zkRoot, String siteId) {
```

Comment:
I think pass the lock path to utility class directly will be better since utility class does not need to know how to make the lock path.
How to define the lock path is decided by applications